README.md

```diff
@@ -2,8 +2,7 @@
 ## Utilize Kubernetes Service-Catalog to dynamically provision CNS Object Storage.
 
 ## WARNING!
-The work in this project is a proof of concept and not intended for anything
-more than that.
+The work in this project is a proof of concept and not intended production.
 
 ## Overview
```

Comment:
Considering adding a **Limitations** section where we can cover multi-tenancy, nested buckets, limits based on the minio client, etc...